narwhals/_pandas_like/series.py

```diff
@@ -174,6 +174,25 @@ def dtype(self: Self) -> DType:
             self._native_series, self._dtypes, self._implementation
         )
 
+    def ewm_mean(
+        self,
+        *,
+        com: float | None = None,
+        span: float | None = None,
+        half_life: float | None = None,
+        alpha: float | None = None,
+        adjust: bool = True,
+        min_periods: int = 1,
+        ignore_nulls: bool = False,
```

Comment:
there's a lot of parameters here, do we have a test which hits each of them?

Reply:
I'll add tests for the parameters then... (I can't hit all the parameters in only one test, at least not the first 4 I think).. Is that what you meant?